webapp/src/components/common/Matrix/shared/utils.ts

```diff
@@ -33,6 +33,31 @@ import { getCurrentLanguage } from "@/utils/i18nUtils";
 import { Aggregate, Column, TIME_FREQUENCY_CONFIG } from "./constants";
 import { groupHeaderTheme } from "../styles";
 
+/**
+ * Checks if a string value is a valid numeric representation that can be parsed.
+ *
+ * @param value - The value to check
+ * @returns true if the string can be converted to a valid number, false otherwise
+ */
+export function isNumericString(value: string | number): boolean {
```

Comment:
`isNumeric` more appropriate, because the function allow number value.

`isNumericString(1)` returns `true` but it's not a numeric string.

`: boolean` can be removed.